library/tiqr/Tiqr/OATH/OCRAWrapper_v1.php

```diff
@@ -63,7 +63,8 @@ public function generateChallenge()
      */
     public function generateSessionKey() 
     {
-        return Tiqr_Random::randomHexString(self::SESSIONKEY_SIZE);
+        // TODO: Handle exception
```

Comment:
As a reviewer, seeing a TODO comment always triggers an alarm. 

1. Should this todo be resolved in this PR?
2. What should be done exactly in the future if we choose to leave this statement here? Assuming the application does not catch exceptions, having the exception bubble up into the application would probably result in a 500 error in the Tiqr GSSP somewhere. A log message containing `random_bytes did not return the requested number of bytes` would end up in the logs. Maybe that is good enough for now?

Reply:
Good point. I tried to address this point in the PR text after I wrote this by:
> Note: Although unlikely random_bytes() can throw, which means that tiqr-server-libphp functions that previously did not throw can now throw an exception

@1: I think there is a lot of work "TODO" here. The problem IMO is that the library is inconsistent in how it deals with (Exceptional) error conditions. How a function deals with it is not described in the interface most of the time. A few functions in the library throw, other return false, NULL or an empty string. Some functions fail silently.

The functions that might throw now are the "get some sort of session key/secrer/nonce" functions. Which is fine. It is "Stepup-tiqr" (and other users of the library) that must expect exceptions.

@2: That message is specific enough IMO, but I'm biased ;)

Thinking about it I propose I remove the TODO and handle this in the upcoming sprint. I made an explicit story https://www.pivotaltracker.com/story/show/181622414